src/serlo-editor/plugins/image/index.ts

```diff
@@ -83,5 +86,5 @@ export type ImageProps = EditorPluginProps<ImagePluginState, ImageConfig>
 
 export interface ImagePluginConfig {
   upload: UploadHandler<string>
-  validate: UploadValidator
+  validate: UploadValidator<Array<{ errorCode: number; message: string }>>
```

Comment:
There is this [FileError interface](https://github.com/serlo/frontend/blob/staging/src/serlo-editor/plugins/image/image-with-serlo-config.ts#L39) defined already, which you could reuse here. What do you think?

If you do reuse it, please [import it as a type](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export), to avoid circular dependency issues.

Reply:
Fixed. Thanks for pointing this out!